### PR TITLE
Footer created

### PR DESCRIPTION
**Pull request by AnitaMartinez**
Lorem ipsum...

### CODE REVIEW

**Review by kendaltw — CHANGES_REQUESTED** [review]
Color needs to be green, instead of red. Thanks in advance (:

**Review by Tania-nad — COMMENTED** [review]
The code is correct, good job!

File: css/styles.css [hunk 1]
```diff
@@ -17,3 +17,12 @@ button {
...
   background-color: lightblue;
 }
+
+.footer {
+  display: flex;
+  justify-content: center;
+}
+
+a {
+  color: red;
```
Comment (kendaltw) [inline css/styles.css]: is this where i need to comment about the color?

Comment (mee96) [inline css/styles.css]: the color should be green, thanks

File: index.html
Comment (ANGELACAMU) [inline index.html:1]: Keep going your are doing great!

**Review by RaquelCosta07 — APPROVED** [review]
Well done!

File: css/styles.css (hunk 1 above)
Comment (Marta-ms) [inline css/styles.css]: the correct color is blue

File: css/styles.css [hunk 2]
```diff
@@ -17,3 +17,12 @@ button {
 .header {
   background-color: lightblue;
 }
+
+.footer {
+  display: flex;
+  justify-content: center;
+}
+
+a {
```
Comment (Crisferdi8) [inline css/styles.css:26]: is thats ok!

File: css/styles.css [hunk 3]
```diff
@@ -17,3 +17,12 @@ button {
 .header {
   background-color: lightblue;
 }
+
+.footer {
```
Comment (Bhetleem) [inline css/styles.css:21]: I would add space between style. What do you think?

**Review by Crisferdi8 — APPROVED** [review]

File: css/styles.css [hunk 4]
```diff
@@ -17,3 +17,12 @@ button {
 .header {
   background-color: lightblue;
 }
+
+.footer {
+  display: flex;
+  justify-content: center;
```
Comment (Nuria8890) [inline css/styles.css:23]: You could add align-items

**Review by sprietoviciana — APPROVED** [review]

**Review by Tania-nad — APPROVED** [review]

**Review by kendaltw — CHANGES_REQUESTED** [review]

**Review by Bhetleem — COMMENTED** [review]
I would add one more style to the footer. Thanks :-)

**Review by ANGELACAMU — COMMENTED** [review]
You are doing great! keep going!

File: css/styles.css (hunk 1 above)
Comment (AnitaMartinez) [reply css/styles.css]: That's true, I had forgotten the last meeting with design. 
https://github.com/AnitaMartinez/clase-git-colaborativo/pull/2/commits/8ba1a3aab4d1df8fee9c150820942743fcc02d64

Comment (mee96) [reply css/styles.css]: That's perfect <3

**Pull request merged**